pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -247,9 +247,50 @@ func (c *Cluster) failMon(monCount, desiredMonCount int, name string) {
 	}
 }
 
+func (c *Cluster) enableMonDaemon(name string, enabled bool) error {
```

Comment:
I'm confused by the name of this method. Should it be something like `updateMonDepReplica`?

Reply:
Yes, I'll rename it for clarity